components/Navbar/Navbar.tsx

```diff
@@ -41,19 +27,21 @@ function Search() {
   );
 
   return (
-    <SearchWrapper searchFocused={searchOpened}>
-      <SearchInput
+    <Styled.SearchWrapper>
+      <Styled.SearchInput
+        searchFocused={searchOpened}
         type="text"
         placeholder="Search"
         onFocus={() => setSearchOpened(true)}
         onBlur={() => setSearchOpened(false)}
         value={input}
         onInput={(e: React.ChangeEvent<HTMLInputElement>) => setInput(e.target.value)}
       />
-      <button type="submit" onClick={search}>
-        Go
-      </button>
-    </SearchWrapper>
+      <Styled.SearchButton type="submit" onClick={search}>
+        <Image src="/assets/icons/icon-search.svg" width="16" height="16" alt="Icon for search" />
```

Comment:
Lets use a different aproach for images.
`import LogoImage from "../../public/assets/icons/someImage.svg";`
```
<Image
              className="img-logo"
              src={LogoImage.src}
              width={LogoImage.width}
              height={LogoImage.height}
              alt="Orange Logo"
            />
```
Same as for Orange Logo